msal/src/main/java/com/microsoft/identity/client/internal/controllers/LocalMSALController.java

```diff
@@ -457,4 +458,15 @@ public boolean getDeviceMode(CommandParameters parameters) throws Exception {
     public boolean removeCurrentAccount(RemoveAccountCommandParameters parameters) throws Exception {
         return removeAccount(parameters);
     }
+
+    // Placeholder methods to avoid inheritance error (these methods will be declared in BaseController)
+    // Also why the @Override annotation is missing
+    public AuthorizationResult deviceCodeFlowAuthRequest(DeviceCodeFlowCommandParameters parameters) throws Exception {
+        // TODO: Placeholder to avoid inheritance error. Will be implemented after Command/Controller level PR in Common
+        return null;
+    }
+    public AcquireTokenResult acquireDeviceCodeFlowToken(AuthorizationResult authorizationResult) throws Exception {
+        // TODO: Placeholder to avoid inheritance error. Will be implemented after Command/Controller level PR in Common
+        return null;
+    }
```

Comment:
Same feedback as given above for `BrokerMsalController`

Reply:
+1. Should be declared in BaseController